std_msgs/Header header

float32 pos_x
float32 pos_y